node/__tests__/validation.spec.ts

```diff
@@ -0,0 +1,164 @@
+import { validation } from '../utils/validation'
+
+describe('Test valdiation', () => {
+  it('Test if validation have been called with type = ""', async () => {
+    const validationValues = {
+      type: '',
+      content: 'Aroma',
+      name: 'Aromas',
+      simpleStatements: [
+        {
+          object: {
+            name: '3',
+            value: '4',
+          },
+          subject: 'specificationProperties',
+          verb: '=',
+        },
+      ] as unknown as [],
+      operator: 'all',
+      priority: 1,
+    }
+
+    expect(() => {
+      validation(validationValues, false)
+    }).toThrow('It is necessary to send a type')
+  })
+
+  it('Test if validation have been called with name = ""', async () => {
+    const validationValues = {
+      type: 'html',
+      content: 'Aroma',
+      name: '',
+      simpleStatements: [
+        {
+          object: {
+            name: '3',
+            value: '4',
+          },
+          subject: 'specificationProperties',
+          verb: '=',
+        },
+      ] as unknown as [],
+      operator: 'all',
+      priority: 1,
+    }
+
+    expect(() => {
+      validation(validationValues, false)
+    }).toThrow('It is necessary to send a name')
+  })
+
+  it('Test if validation have been called with content = ""', async () => {
+    const validationValues = {
+      type: 'html',
+      content: '',
+      name: 'Aromas',
+      simpleStatements: [
+        {
+          object: {
+            name: '3',
+            value: '4',
+          },
+          subject: 'specificationProperties',
+          verb: '=',
+        },
+      ] as unknown as [],
+      operator: 'all',
+      priority: 1,
+    }
+
+    expect(() => {
+      validation(validationValues, false)
+    }).toThrow('It is necessary to send a content')
+  })
+
+  it('Test if validation have been called with simpleStaments = []', async () => {
+    const validationValues = {
+      type: 'html',
+      content: 'Aromas',
+      name: 'Aromas',
+      simpleStatements: [] as unknown as [],
+      operator: 'all',
+      priority: 1,
+    }
+
+    expect(() => {
+      validation(validationValues, false)
+    }).toThrow('It is necessary to send a simpleStatement')
+  })
+
+  it('Test if validation have been called with operator = ""', async () => {
+    const validationValues = {
+      type: 'html',
+      content: 'Aromas',
+      name: 'Aromas',
+      simpleStatements: [
+        {
+          object: {
+            name: '3',
+            value: '4',
+          },
+          subject: 'specificationProperties',
+          verb: '=',
+        },
+      ] as unknown as [],
+      operator: '',
+      priority: 1,
+    }
+
+    expect(() => {
+      validation(validationValues, false)
+    }).toThrow('It is necessary to send a operator')
```

Comment:
```suggestion
    }).toThrow('It is necessary to send an operator')
```

(I think you'll have to change the message where the exception is originally generated as well)